Description: port to newer jQuery
 replace usage of $.live()
Author: Antonio Terceiro <terceiro@debian.org
Origin: vendor
Forwarded: no
Last-Update: 2015-10-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/assets/javascripts/application.js
+++ b/assets/javascripts/application.js
@@ -91,10 +91,10 @@ $(document).ready(function() {
   });
 
   // Make sure tabs don't get ugly focus borders when active
-  $('.group_tabs a').live('focus', function() { $(this).blur(); });
+  $('body').on('focus', '.group_tabs a', function() { $(this).blur(); });
 
   var favicon_path = $('link[rel="shortcut icon"]').attr('href');
-  $('.group_tabs a').live('click', function(){
+  $('body').on('click', '.group_tabs a', function(){
     if (!$(this).parent().hasClass('active')) {
       $('.group_tabs a').parent().removeClass('active');
       $(this).parent().addClass('active');
